stdlib/src/os/os.mojo

```diff
@@ -186,7 +186,16 @@ struct _DirHandle:
 # ===----------------------------------------------------------------------=== #
 # listdir
 # ===----------------------------------------------------------------------=== #
-fn listdir(path: String = "") raises -> List[String]:
+fn listdir() raises -> List[String]:
+    """Gets the list of entries contained in the empty path.
+
+    Returns:
+      Returns the list of entries in the path provided.
+    """
+    return listdir("")
+
+
+fn listdir(path: String) raises -> List[String]:
```

Comment:
Should we change the signature to `fn listdir(path: String) raises -> List[String]` and avoid introducing `fn listdir() raises -> List[String]`?